pkg/codegen/python/doc.go

```diff
@@ -47,6 +47,11 @@ func (d DocLanguageHelper) GetDocLinkForFunctionInputOrOutputType(packageName, m
 	return ""
 }
 
+// GetDocLinkForBuiltInType returns the godoc URL for a built-in type.
+func (d DocLanguageHelper) GetDocLinkForBuiltInType(typeName string) string {
+	return ""
```

Comment:
What do you think about creating a switch-case expression here to return the appropriate link to the [built-in types](https://docs.python.org/3/library/stdtypes.html) in Python? For example, `string` type in our schema would link to [`str`](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str).

Reply:
I went ahead and added a link for the built in types. Seeems like there is only one top-level page for all built in types similar to how the .NET docs are structured, so I just return the link like we do there.